### DISCUSSION TITLE
Disabled links are focusable

### DISCUSSION THREAD

**Discussion by ritz078** [adobe/react-spectrum#4903] [Q&A] [answered]
It looks like keeping the links focusable even if they are disabled is an expected behaviour. It makes sense but is there any documentation/guideline based on which this decision was taken because for other disabled components like button, focus is not allowed.

**Comment by LFDanLu** [comment]
I can't find any recorded hard guidelines or discussions the team may have had regarding this from years past, but for buttons we followed the native behavior where disabled buttons aren't focusable. Native links don't have the same kind of disabled support so we didn't have the same kind of precedence to follow there. One thing to note for the disabled link case would be that a disabled link would be almost indistinguishable from text if it wasn't focusable and didn't have any additional styling so there is some merit there in making them focusable while disabled.

**Reply by snowystinger** [reply, marked as answer]
Relevant issues around disabled+focus:
https://github.com/adobe/react-spectrum/issues/3662
https://github.com/adobe/react-spectrum/issues/1696

**Reply by ritz078** [reply]
Thank you both for the explanation. I can see that there's progress on this front which is great.
